packages/yoga/src/Theme/themes/Corporate.js

```diff
@@ -2,8 +2,8 @@ import BaseTheme from './BaseTheme';
 
 const Corporate = tokens => {
   const colors = {
-    primary: tokens.colors.newYork,
-    secondary: tokens.colors.paris,
+    primary: tokens.colors.buenosAires,
+    secondary: tokens.colors.newYork,
```

Comment:
@tsuzukayama talking with the product designers, we figured that the secondary color pallete for corp is actually `saoPaulo`

Reply:
@alan-oliv I updated the `buenosAires` and `saoPaulo` colors. Do you mind taking a look? There were some snapshot updates too, since `saoPaulo` is used on other places.